### ISSUE TITLE
Release 2.1.0 not compatible with oxid-esales/oxideshop-ce:6.0.0

### ISSUE THREAD

**Issue by kaluzki** [OXID-eSales/ddoe-wysiwyg-editor-module#8]
```
( ! ) Fatal error: Call to undefined method OxidEsales\WysiwygModule\Application\Controller\TextEditorHandler::isTextEditorDisabled()
```

The commit 
https://github.com/OXID-eSales/oxideshop_ce/commit/bbcd3e8181dcd04c7c005ec2a1a3e9f1fd81b88f#diff-2dbde040f56515ba07151d83716b8f55 needs to be released

**Comment by MantasVaitkunas** [comment]
Hi @kaluzki,
thanks for informing about the issue. This release is compatible only with upcoming compilation version, because of these changes we had to increase major module version and not a minor, but since it's already done I have updated README.md by adding requirements section - https://github.com/OXID-eSales/ddoe-wysiwyg-editor-module/commit/1c50117eb560696924dd2d77bfe9025a6b30aeca.
I'll leave this issue open until the compilation v6.0.1 release.
